.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Report a bug or an issue you've found with Trino Python client
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+### Describe the bug
+A clear and concise description of what the bug is. What script did you run? What happened?
+
+### Steps To Reproduce
+In as much detail as possible, please provide steps to reproduce the issue. Sample code that triggers the issue, relevant server settings, etc is all very helpful here.
```

Comment:
I think this should be combined with the last section, `System information`. This is so important that maybe even adding a example would be nice, like:
> Using `trino-python-client version X connect to Trino version Y using the connection string 'trino://user@host.example:8080' and see the error message

or
> Using `trino-python-client version X execute the following query on a Trino server version Z:
> ...query and sample data

Reply:
The advantage of separating it is that it would ensure we have a minimum of information, while free text or an example would shift this responsibility to the user.

Reply:
But with just server version or client version or OS you can't debug anything - you can guess what went wrong.
So having just System Information without steps is useless.

Reply:
Sure I'm suggesting to keep both sections actually.